### PR TITLE
Guests popup

### LINKED ISSUE #73: Hotels: guests popup
Please try to create guests pop up in hotels view. It may be a little bit complicated to do the grey overlay correctly - we will see.

screen shot 2017-12-20 at 11 27 28

https://zpl.io/VOX1roX

### PR DESCRIPTION
**Pull request by jakubzaba**
Closes #73

Children are not defined yet. Want to review this PR and children finish in different PR?

### CODE REVIEW

**Comment by mrtnzlml** [comment]
Please test these changes in [Expo](https://docs.expo.io/versions/latest/introduction/installation.html#mobile-client-expo-for-ios-and-android) after build has finished:

  QR Code

**Comment by mrtnzlml** [comment]
I tried it and after I close the popup I can sometimes see little "blink" of the whole screen. Can you please check what's going on? It has very annoying effect...

**Comment by jakubzaba** [comment]
@mrtnzlml Children are not defined yet. Want to review this PR and children finish in different PR?

**Comment by VaclavSir** [comment]
There should be always at least 1 adult. Now you can enter 0 or even a negative value.

**Comment by jakubzaba** [comment]
Thanks, I knew there was sth I wanted to do for this PR :) Fixed.

File: app/common/src/forms/NumberControl.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  );
+}
+
+const styles = StyleSheet.create({
+  control: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  icon: {
+    color: '#78818a',
```
Comment (mrtnzlml) [inline app/common/src/forms/NumberControl.js]: Did you mean `Color.icon.grey`? 🙄

**Comment by mrtnzlml** [comment]
Integrated and umbrella issues opened.

**Pull request closed without merging**